tests/test_modeling_common.py

```diff
@@ -2708,7 +2708,7 @@ def test_load_with_mismatched_shapes(self):
     def test_model_is_small(self):
         # Just a consistency check to make sure we are not running tests on 80M parameter models.
         config, _ = self.model_tester.prepare_config_and_inputs_for_common()
-        # print(config)
+        print(config)
```

Comment:
guess you will remove it before merge :-)

Reply:
Removed!